source/_integrations/openweathermap.markdown

```diff
@@ -18,7 +18,7 @@ ha_platforms:
 ha_integration_type: integration
 ---
 
-The OpenWeatherMap weather integrations uses [OpenWeatherMap](https://openweathermap.org/) as a source for current meteorological data for your location.
+The OpenWeatherMap weather integrations uses [OpenWeatherMap](https://openweathermap.org/) as a source for current meteorological data for your location. It uses the [Weather integration platform](/integrations/weather/).
```

Comment:
That is an odd sentence to write, we don't write such things anywhere. Additionally, it uses more entity components, not just the `weather` one.

```suggestion
The OpenWeatherMap weather integrations uses [OpenWeatherMap](https://openweathermap.org/) as a source for current meteorological data for your location.
```

Reply:
I will rethink this. I wanted a way to link back to the weather integration so people can see the common information like the forecast details.

Reply:
Inntegration can either list information on what they provide, some add chapters on their sensors and other entities.

A more general solution would be leveraging the `ha_platforms` frontend matter, to automatically list them in the aside maybe...